src/roll_renderer.pyj

```diff
@@ -462,6 +462,12 @@ class Beyond20RollRenderer:
                 elif request.name == "Toll the Dead":
                     ttd_dice = v'await self.queryGeneric(request.name, "Is the target missing any of its hit points?", {"d12": "Yes", "d8": "No"}, "ttd_dice", ["d12", "d8"])'
                     damages[0] = damages[0].replace("d8", ttd_dice)
+                elif damage_types.includes("Colossus Slayer"):
+                    cs_index = damage_types.indexOf("Colossus Slayer")
+                    cs_dice = ttd_dice = v'await self.queryGeneric(request.name, "Is the target missing any of its hit points?", {"d8": "Yes", "0": "No"}, "ttd_dice", ["d8", "0"])'

```

Comment:
Other than the prompt which I suggest you to change it to : 
```
Add Colossus Slayer damage?
(It can only be applied once per turn, if enemy has sustained damage)
```
There's also that you did `cs_dice = ttd_dice = ` which should just be `cs_dice` and the 3rd argument to `queryGeneric` is "ttd_dice" when it should be "cs_dice".